src/components/DeployGuidesNav.astro

```diff
@@ -23,6 +23,7 @@ interface Service {
 const services: Service[] = [
 	{ title: 'Netlify', slug: 'netlify', supports: ['ssr', 'static'] },
 	{ title: 'Vercel', slug: 'vercel', supports: ['ssr', 'static'] },
+	{ title: 'AWS via Flightcontrol', slug: 'flightcontrol', supports: ['ssr', 'static'] },
```

Comment:
We're thinking about where in this list to put you! 😄 

Because we do have the filtering options, you're not going to get lost among the other SSR options. And, we admittedly have been usually adding new options to the bottom of the list, having sponsors and providers we know a lot of our users are currently using near the top. 

I think maybe closer to the Cloudflare/Heroku sectiion is more reasonable, and we'd probably move AWS up closer to you. But, I'm OK to think about this a bit more and maybe even make a change later. I guess this is more of a heads up not to be surprised if this gets reordered!